iterations_runners/iterations_runner.c

```diff
@@ -99,14 +111,20 @@ main(int argc, char **argv)
         }
 
         /* timed section */
-        start_time = clock_gettime_monotonic();
+        krun_tsr_start = read_ts_reg();
```

Comment:
It would be better to do:

```c
read_clock();
read_tsr();
do_bench();
read_tsr();
read_clock();
```

so that the TSR doesn't include the clock. [The clock resolution is waaaaaaay below the TSR resolution, which is why I'm suggesting this way around.]